.travis.yml

```diff
@@ -51,6 +51,8 @@ install:
     - pip install -U pip wheel setuptools
     # Install AiiDA with some optional dependencies
     - pip install .[REST,docs,atomic_tools,testing]
+    # Unreleased plum version required for fast-forwarding
```

Comment:
This should be removed before merging, after creating a patched ``plum`` 0.7.x release.